src/include/mutex.h

```diff
@@ -74,6 +74,10 @@ struct __wt_rwlock {
 
 struct WT_COMPILER_TYPE_ALIGN(WT_CACHE_LINE_ALIGNMENT) __wt_spinlock {
 	volatile int lock;
+
+	int16_t stat_count;		/* acquisitions */
+	int16_t stat_app_usecs;		/* waiting application threads */
+	int16_t stat_int_usecs;		/* waiting server threads */
```

Comment:
It took me a while to figure out these were the offset for the stat field rather than collecting the actual usecs value.  Can the comments and/or the fields add "offset of" or something similar?